src/components/Explore/Explore.tsx

```diff
@@ -252,13 +255,17 @@ const Explore: React.FunctionComponent<{
       }
     }, [pathname, scrollToExplore]);
 
-    // if the pathname changes (ie. if navigating between location pages via compare or regionmap)-
-    // resets metric, time period, and locations
-    // (need to force the reset since the route doesnt change)
+    const defaultTimePeriod = useDefaultTrendsPeriod();
+
+    // Resets time period state variable when pathname changes (need to force the reset since the route doesn't change):
+    useEffect(() => {
+      setPeriod(defaultTimePeriod);
+    }, [pathname, defaultTimePeriod]);
+
+    // Resets locations+current metric state variables when pathname changes (need to force the reset since the route doesn't change):
     useEffect(() => {
       setSelectedLocations(initialLocations);
       setCurrentMetric(ExploreMetric.HOSPITALIZATIONS);
-      setPeriod(Period.ALL);
```

Comment:
need to separate this out into its own effect, so a change in screen size doesn't reset the selected metric+locations